src/javascript/app/pages/user/view_popup/view_popup.js

```diff
@@ -120,24 +121,15 @@ const ViewPopup = (() => {
             RUNLOW      : localize('Only Downs'),
             MULTUP      : localize('Multiplier Up'),
             MULTDOWN    : localize('Multiplier Down'),
-        });
-
-        return {
-            get: () => {
-                if (!contract_type_display) {
-                    contract_type_display = initContractTypeDisplay();
-                }
-                return contract_type_display;
-            },
```

Comment:
You need to follow the standard for the project and this is how we define the basic functionalities of each component. you need to revert this back.
Also it is breaking other places in which `get` method is being used.

Reply:
Same as the other pull request for DSmartTrader 🙏.